docs/import.md

```diff
@@ -144,9 +144,19 @@ ollama push <your username>/example
 
 After publishing, your model will be available at `https://ollama.com/<your username>/example`.
 
+### Tagging models
+
+If your model has different versions you can optionally tag them, without a tag, it will be `latest`:
+
+```
+ollama push <your username>/example[:tag]
```

Comment:
Is this an OK example? It's based on Docker's [push reference](https://docs.docker.com/reference/cli/docker/image/push/). Alternatively, the brackets can be removed.

```suggestion
ollama push <your username>/example:tag
```